src/components/PrintModal/PrintModal.js

```diff
@@ -41,20 +42,29 @@ class PrintModal extends React.PureComponent {
     this.includeComments = React.createRef();
     this.pendingCanvases = [];
     this.state = {
+      allowWatermarkModal: false,
```

Comment:
What's this state used for? If an user has programmatically called `setWatermark` then we don't want to show the edit watermark option?

Reply:
correct.